build.gradle

```diff
@@ -42,13 +32,6 @@ allprojects {
     }
 
     apply plugin: 'org.inferred.processors'
```

Comment:
we should make errorprone plugin just apply this - it's silly to have this explicitly defined just for errorprone